src/styles/watch.scss

```diff
@@ -0,0 +1,46 @@
+.watch {
```

Comment:
Your stopwatch should be in the center of the page (3-rd point in the task)

Reply:
it is in the center thanks to margin 0 auto